src/intervals/arithmetic.jl

```diff
@@ -375,6 +375,9 @@ end
     cancelminus(a, b)
 
 Return the unique interval `c` such that `b+c=a`.
+
+See Section 12.12.5 of the IEEE-1788 Standard for
+Interval Arithmetic.
 """
 function cancelminus(a::Interval, b::Interval)
     T = promote_type(eltype(a), eltype(b))
```

Comment:
This can be written (I believe)
```jl
cancelminus(a::Interval, b::Interval) = cancelminus(promote(a, b))

function cancelminus{T}(a::Interval{T}, b::Interval{T})
   ...
end
```